server/db/mysql/adapter.go

```diff
@@ -823,12 +823,12 @@ func (a *adapter) UserGetAll(ids ...t.Uid) ([]t.User, error) {
 	if err != nil {
 		return nil, err
 	}
+	defer rows.Close()
 
 	var user t.User
 	for rows.Next() {
-		if err = rows.StructScan(&user); err != nil {
-			users = nil
-			break
+		if err := rows.StructScan(&user); err != nil {
```

Comment:
The `err` is already allocated. There is no value in allocating it here again. Please use `err =` instead of `err :=`.

Reply:
There is value. It keeps the variable scope smaller.